<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlotStoreRenameDialog</class>
 <widget class="QDialog" name="SlotStoreRenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dateienpfad umbenennen</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="description">
      <property name="text">
       <string>Sucht in der Spalte &quot;Datei&quot; in ALLEN Einträgen und ersetzt den Text oben durch den Text unten. (Groß-/Kleinschreiben wird unterschieden!)</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>SUCHEN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="replaceWhatField"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ERSETZEN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="replaceWithField"/>
    </item>
    <item>
     <widget class="QCheckBox" name="replaceSlotsCheckBox">
      <property name="text">
       <string>Auch in den Layern ersetzen (LANGSAM!)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SlotStoreRenameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SlotStoreRenameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
